python/dgl/data/__init__.py

```diff
@@ -76,6 +76,7 @@
 # datasets.
 try:
     from .lrgb import (
+        COCOSuperpixelsDataset,
         PeptidesFunctionalDataset,
         PeptidesStructuralDataset,
         VOCSuperpixelsDataset,
```

Comment:
I refer to the original code here https://github.com/vijaydwivedi75/lrgb/blob/main/graphgps/loader/dataset/coco_superpixels.py

Reply:
I see, but usually the name in the paper is regarded as a unique id to find them. Maybe add original names in the documentation so that users can find these datasets with original names as queries.